### PR TITLE
Update 11PageTitles.md

### PR DESCRIPTION
**Pull request by cliffbert**
Since step1 test is "Check that a page <title> element is defined for the page. [SC 2.4.2]", the IC should be the page itself. Probably like 15.1: "Pages with text (including text alternatives)."

### CODE REVIEW

File: 11PageTitles.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Limitations, Assumptions, Exceptions
 ---------------------------------------
 **Baseline Test ID:** 11.1-PageTitled
 ### Identify Content
-<p id="1IC">Page <code>&lt;title&gt;</code> element for the page.</p>
+<p id="1IC">HTML web page.</p>
```
Comment (kengdoj) [inline 11PageTitles.md:22]: agree with the intent of change... should it just be "web page"?

Comment (cliffbert) [reply 11PageTitles.md:22]: Seems right

**Review by kengdoj — APPROVED** [review]
Not sure why all these other pages are part of this PR, but they all look ok.

**Comment by kengdoj** [comment]
With so many conflicting files, it may be easiest to redo this change in a newer branch.

**Comment by kengdoj** [comment]
change to 11PageTitles Identify Content made.

**Pull request closed without merging**